src/types/global.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'lucide-react';
+declare module 'mermaid';
```

Comment:
Why need this type declaration, can you show what type error you are trying to fix?